shlr/sdb/src/ls.c

```diff
@@ -45,7 +45,7 @@ static SdbListIter *_merge(SdbListIter *first, SdbListIter *second, SdbListCompa
 		} else if (!first) {
 			next = second;
 			second = second->n;
-		} else if (cmp (first->data, second->data) < 0) {
+		} else if (cmp (first->data, second->data) <= 0) {
```

Comment:
@xvilka this should not be merged here, it should be changed in sdb code base first and then sdb needs to be updated in r2.

Reply:
You mean two different commits instead of updating both at the same time like here ?

Reply:
Sdb is actually taken from another repo, see https://github.com/radareorg/sdb . Changes to sdb files should first be merged there and then you can use `make sdb-sync` to update sdb

Reply:
I sent a separate PR for sdb and removed this.